internal/provisioner/plain/controllers/bundle_controller.go

```diff
@@ -148,7 +152,11 @@ func (r *BundleReconciler) Reconcile(ctx context.Context, req ctrl.Request) (_ c
 
 	pod := &corev1.Pod{}
 	if op, err := r.ensureUnpackPod(ctx, bundle, pod); err != nil {
-		u.UpdateStatus(updater.SetBundleInfo(nil), updater.EnsureBundleDigest(""))
+		u.UpdateStatus(
+			updater.SetBundleInfo(nil),
+			updater.EnsureBundleDigest(""),
+			updater.EnsureContentURL(""),
+		)
```

Comment:
Out of scope for this PR, but I wonder if we should _always_ set these fields this way when unpacking is failing (i.e. move these to `updateStatusUnpackFailing()`. I'm trying to remember if there was a case where you'd want to keep these fields set from a previous unpack, but still update the conditions and phase to say "Failing" based on the current unpack.

Now that we have bundle immutability, I'm tempted to consolidate these and always unset them when an unpack is failing.

I'm